05.ls/ls.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+COLUMN_LENGTH = 3
+
+def main
+  max_filename_char_length = count_filename_char
+
+  prepare_files_to_display.each do |files|
+    files.each do |file|
+      print file.ljust(max_filename_char_length + 1) if file
+    end
+    print "\n"
+  end
+end
+
+def load_files
+  Dir.entries('.').sort.grep_v(/^\./)
+end
+
+def count_filename_char
+  load_files.max_by(&:length).length
+end
+
+def prepare_files_to_display
+  non_dot_files = load_files
```

Comment:
```suggestion
  files = load_files
```

non dotであるかどうかはこの処理ではあまり関係ないので変数名に含めなくてもいいと思いますー